pandas/io/formats/html.py

```diff
@@ -306,6 +306,8 @@ def _column_header():
             align = self.fmt.justify
 
             if truncate_h:
+                if self.fmt.index is False:
```

Comment:
Instead of doing `is False` can you leverage the implicit truthiness here and do `if not self.fmt.index`? While not documented I believe `index=0` and `index=None` are acceptable in this and other parsers, so would be ideal to handle those consistently with False

Reply:
can you update this